lib/takeaway.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'menu'
+require 'twilio-ruby'
+
+class Takeaway
+
+  ACCOUNTS_SID = ENV['TWILIO_SID']
+  AUTH_TOKEN = ENV['TWILIO_AUTH']
+
+  attr_reader :menu
+
+  def initialize(menu_klass = Menu.new, twilio_klass = Twilio::REST::Client.new(ACCOUNTS_SID, AUTH_TOKEN))
```

Comment:
`menu_class` is relating to an instance, not a class. IT would have been better to use `menu = Menu.new, twilio_client = Twilio::blablabla`